### DISCUSSION TITLE
Possible mismatched Windows power modes (G14 2022, Win 11 22H2)

### DISCUSSION THREAD

**Discussion by senoj26** [seerge/g-helper#1712] [Q&A] [answered]
I have a G14 2022 with Windows 11 version 22H2 (build 22621.2715) installed and G-Helper 0.134.0.

The README shows how the three CPU power modes in G-Helper correspond to the Windows power modes here:
<img width="365" alt="image" src="https://github.com/seerge/g-helper/assets/20372625/d70eaf6f-1386-4239-8784-ef2a959d79b8">

However, my system has four power modes:
<img width="268" alt="image" src="https://github.com/seerge/g-helper/assets/20372625/96fb323d-fc40-4fb9-bca3-99814dd6be28">

Does anyone know how those correspond to the ones in G-Helper? Could this potentially limit performance if set incorrectly? Maybe this warrants submitting an issue so G-Helper can better interface with systems with 4 Windows modes?

**Comment by Kikimahe** [comment]
Hi,

I think you are confusing Power Plan and Power mode. ;-)

You can have multiple Windows power plans.
(In the Power Plan it is recommended to stay on “balanced”.)
![02](https://github.com/seerge/g-helper/assets/142881210/d84ef44a-1354-45d8-9804-7d1aa5580072)

... But there are only 3 Windows power modes.
![01](https://github.com/seerge/g-helper/assets/142881210/ec48060c-1a31-4237-bef1-a8b7b1b53044)


Ghelper switches between the 3 power modes. :)

**Comment by seerge** [comment, marked as answer]
@senoj26 hello, in Windows 10 it's indeed 4 power modes, and in Windows 11 Microsoft has simplified them to 3 :) 

For simplicity G-Helper allows you to select (under ``Fans+Power`` all 3 of them https://learn.microsoft.com/en-us/windows/win32/power/power-policy-settings)

Windows 10 has one more, that is somewhere in-between, but I don't think that it is really different from ones available :)

**Comment by senoj26** [comment]
Thanks all for your replies. Turns out a third-party application I use that replaces taskbar flyouts with their Win 10 equivalents was causing the confusion. When clicking the battery in the taskbar, the Windows 10 flyout was incorrectly shown with the old 4-mode style.

I confirmed within Settings that there are only 3 power modes and G-Helper switches between them correctly, so fully user error on this one :/
